openslides/agenda/projector.py

```diff
@@ -24,20 +54,25 @@ class ItemDetailSlide(ProjectorElement):
     """
     Slide definitions for Item model.
 
-    To activate this slide as list of speakers slide, set 'list_of_speakers'
-    true.
+    This is only for detail slides. You have to set 'id'.
+
+    To activate a detail slide as list of speakers slide, set 'id' and set
+    'list_of_speakers' true.
     """
-    name = 'agenda/item'
+    name = 'agenda/item-detail'
 
     def get_context(self):
         pk = self.config_entry.get('id')
-        if not Item.objects.filter(pk=pk).exists():
-            raise ProjectorException(_('Item does not exist.'))
-        return {'id': pk, 'list_of_speakers': self.config_entry.get('list_of_speakers', False)}
+        if pk is not None:
+            # Detail slide.
+            if not Item.objects.filter(pk=pk).exists():
+                raise ProjectorException(_('Item does not exist.'))
+            return {'id': pk, 'list_of_speakers': self.config_entry.get('list_of_speakers', False)}
```

Comment:
Could you add a comment or explicitly call return None at the end of the function. Because it looks like an error